ansible/patch-nodes-default.yml

```
@@ -0,0 +1,41 @@
+---
+- hosts: all
```

proper linting means playbooks all of a name, so I usually do name: Main
```suggestion
- name: Main
  hosts: all
```